content/trial_getting_started/set_trial_manager/other-deployments.md

```diff
@@ -64,9 +143,11 @@ For more details read the [5.1.1 Release Notes](https://cloudify.co/release-note
 
 {{< param product_name >}} **Community CLI**: [RPM](https://repository.cloudifysource.org/cloudify/5.1.1/ga-release/cloudify-cli-5.1.1-ga.el7.x86_64.rpm)	|	[DEB](https://repository.cloudifysource.org/cloudify/5.1.1/ga-release/cloudify-cli_5.1.1-ga_amd64.deb)	|	[EXE](https://repository.cloudifysource.org/cloudify/5.1.1/ga-release/cloudify-windows-cli_5.1.1-ga.exe)
 
+
+
 ### 5.1.0
 
-v5.1.1 is the latest __major__ release.
+v5.1.0 is the latest __major__ release.
```

Comment:
v5.2.0 is the latest.

Reply:
Yeah, 5.1.0 isn't/shouldn't be a major release.

Theoretically that should be 5.0.0, but realistically 5.0.5